.circleci/config.yml

```diff
@@ -93,12 +99,14 @@ workflows:
   every-four-hours:
     triggers:
       - schedule:
-          cron: "0 1,5,9,13,17,21 * * *"
+          cron: "40 1,5,9,13,17,21 * * *"
```

Comment:
nitpick: */4 work for the hour field? dunno if supported though. I think there's a way to add some wobble to the minutes field too, depending on platform

Reply:
They don't support */4, it would be more elegant.